src/components/Heading/Heading.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Heading from './Heading';
+
+describe('Heading', () => {
+  test('Renders the heading component', () => {
```

Comment:
A good test to add could be to test the correct usage of properties. Or within the property text-color, the dev can not pass another tailwind class.